### DISCUSSION TITLE
GitHub and Azure: We want to hear all of your tips, tricks, and lessons learned as we gear up for an awesome session on pairing Github & Azure at Universe 2024!

### DISCUSSION THREAD

**Discussion by dsanchezcr** [community/community#138285] [Other Feature Feedback, Questions, & Ideas]
GitHub Universe 2024 Session: [DEV1570L - Better together: Tips, tricks, and lessons learned from pairing GitHub and Azure](https://reg.rainfocus.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715354989964001kqMX)

> In this interactive session, Microsoft's David Sanchez, Krystal Folkes, and Suzanne Daniels will dive into the seamless integration between GitHub and Microsoft Azure and reveal how these technologies work together to enhance software development. You'll discover proven tips and tricks learned through real-world stories and practical scenarios. Presenters will also address common challenges, share strategies, and discuss lessons learned. Engage in a collaborative discussion to gain actionable insights and elevate your projects.

Let's get this discussion going! Please share your thoughts: we want to hear all your ideas, experiences, lessons learned and tips & tricks about pairing Azure & GitHub. To get the brainstorming going, just drop your comment here, and share your thoughts or questions.

And if you are going to [GitHub Universe](https://githubuniverse.com), please do come to our session to share your ideas in the small-group discussion format. We can't wait to see you there! ✨ 🚀

**Comment by 6rz6** [comment]
Amazing concept, waiting for more models to be added

Wish-list: 

⚙️ Allow more than 50 API calls per model per day, around 250 is adequate for testing agentic infrastructures using api calls
⚙️ Allow users from TLV to be included in the early access of new models released (for example openai's 01p and 01m which are only available for US east coast power users, we contribute a share of the rnd here as well 🕹)
⚙️ Add Anthropic LLM Models 
⚙️ Add a Text2Voice2Text model like whisper
⚙️ Add a Text2Img model / Stable Diffusion model 

[6rz6](https://github.com/6rz6)
